templates/gcp/fips.html

```diff
@@ -4,354 +4,446 @@
 {% block meta_description %}Ubuntu Pro FIPS for Google Cloud is an Ubuntu Pro image with FIPS certified modules enabled by default for FedRAMP, HIPAA, PCI and ISO compliance. Ubuntu Pro FIPS images are secure and optimised for production on public cloud.{% endblock %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1Djxdzjy78bau8ELM4-MNGoM2YRa_QjpN1xJ4H4GjFyw/edit#heading=h.c6wjbvvp8mcm{% endblock meta_copydoc %}
 
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
-<section class="p-strip u-no-padding--top u-no-padding--bottom">
-  <div class="p-strip--suru-topped">
-    <div class="row u-equal-height">
-      <div class="col-7 col-medium-4">
+<section class="p-section--hero">
```

Comment:
<img width="427" alt="Screenshot 2024-09-03 at 12 10 48 PM" src="https://github.com/user-attachments/assets/9c938eaa-e506-4ae8-a0fe-584362bf2974">

The CTA buttons are missing on hero section

Reply:
They are not on the copy doc, so shouldn't be there

Reply:
oh, I may have been on the wrong doc, thanks

Reply:
https://docs.google.com/document/d/17Rr6Fdrr7xSPi1g2h7GMTztQUGYqz2hdbNUqK0uQihw/edit

Reply:
Can you update the meta copy docs to the correct one?

Reply:
done